src/apify_client/clients/resource_clients/schedule_collection.py

```diff
@@ -111,20 +116,20 @@ async def list(
         Returns:
             ListPage: The list of available schedules matching the specified filters.
         """
-        return await self._list(limit=limit, offset=offset, desc=desc)
+        return await self._list(limit=limit, offset=offset, desc=desc)  # type: ignore
 
     async def create(
-        self,
+        self: ScheduleCollectionClientAsync,
         *,
         cron_expression: str,
         is_enabled: bool,
         is_exclusive: bool,
-        name: Optional[str] = None,
-        actions: Optional[List[Dict]] = None,
-        description: Optional[str] = None,
-        timezone: Optional[str] = None,
-        title: Optional[str] = None,
-    ) -> Dict:
+        name: str | None = None,
+        actions: list[dict] | None = None,  # type: ignore
```

Comment:
why is the type ignore needed here?

Reply:
solved by allowing Any type

Reply:
It is because of the method named `list` in the same module. It breaks `mypy` :confused:. I already created an issue for it - https://github.com/apify/apify-client-python/issues/162.